pool_configs/examples/kmd.json

```diff
@@ -28,6 +28,12 @@
         "ca": ""
     },
 
+    "explorer": {
```

Comment:
Actually, now that I think about it, this really belongs in the coin configs. You should be able to access it via `it.poolsConfigs[pool].coin.explorer` in the template.